sk0-front/src/app/login/login.page.ts

```diff
@@ -0,0 +1,53 @@
+import { Component } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import {
+  FormsModule,
+  FormControl,
+  FormGroup,
+  ReactiveFormsModule,
+  Validators,
+} from '@angular/forms';
+import { IonicModule } from '@ionic/angular';
+import { LoginService } from './login.service';
+import { AlertController } from '@ionic/angular';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.page.html',
+  styleUrls: ['./login.page.scss'],
+  standalone: true,
+  imports: [IonicModule, CommonModule, FormsModule, ReactiveFormsModule],
+})
+export class LoginPage {
+  loginForm = new FormGroup({
+    email: new FormControl('', [Validators.required, Validators.email]),
+    password: new FormControl('', [
+      Validators.required,
+      Validators.pattern(
+        /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d).{6,}$/ //min 6 chars, lower-uppercase letters + number
+      ),
+      Validators.minLength(6),
+    ]),
+  });
+  constructor(
+    private loginService: LoginService,
+    private alertController: AlertController
+  ) {}
+
+  async loginSubmit() {
+    const response = this.loginService.loginrequest(this.loginForm);
+    this.loginForm.reset();
+    if (!response) {
+      let alert = await this.alertController.create({
+        message: 'Nem megfelelő email cím vagy jelszó',
+        buttons: ['OK'],
+      });
+      alert.present();
+    } else {
+      let alert = await this.alertController.create({
+        message: 'Sikeres bejelentkezés',
+      });
+      alert.present();
+    }
+  }
```

Comment:
Usually for better testability we tempt to reset the values last.

The loginService call is usually an async process so should be awaited. 

I would use a ternary operator instead of the to `let` and create a const `alert` based on the response. (Not sure how the response is a falsy-truthy value tho).

Reply:
Becommitoltam, ilyesmire gondoltál?